use-cases/vehicle-routing/src/test/java/org/acme/vehiclerouting/domain/jackson/VRPScoreAnalysisJacksonModule.java

```diff
@@ -0,0 +1,11 @@
+package org.acme.vehiclerouting.domain.jackson;
+
+import ai.timefold.solver.core.api.score.analysis.ScoreAnalysis;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+public class VRPScoreAnalysisJacksonModule extends SimpleModule {
```

Comment:
AFAIK modules can be registered as a resource, so jackson will read them from there.
I suggest we do it like this, to show people the proper way.

See `META-INF/resources` in `timefold-solver/persistence`.

Reply:
Ideally, we would like to show only serialization (no deserialization) of justification objects, but I haven't found how to make it possible with Jackson.

Reply:
Nice, will do!

Reply:
I think showing even the deserialization could be beneficial, users might need to do that on their side anyway so it would be nice to have an example?

Reply:
Let's not overthink this. It's nothing evil, and it's not production code. Ought to be enough.

Reply:
Done.